scripts/exgdas_enkf_update.sh

```diff
@@ -288,7 +275,7 @@ cat > enkf.nml << EOFnml
    univaroz=.false.,adp_anglebc=.true.,angord=4,use_edges=.false.,emiss_bc=.true.,
    letkf_flag=${letkf_flag},nobsl_max=${nobsl_max},denkf=${denkf},getkf=${getkf}.,
    nhr_anal=${IAUFHRS_ENKF},nhr_state=${IAUFHRS_ENKF},
-   lobsdiag_forenkf=${lobsdiag_forenkf},taperanalperts=${taperanalperts},
+   taperanalperts=${taperanalperts},
```

Comment:
An `lobsdiag_forenkf` entry may still be required when writing the EnKF namelist.  It depends on whether the EnKF initializes `lobsdiag_forenkf` to.  I'd suggest just setting it to a constant `.true.`.

```suggestion
    lobsdiag_forenkf=.true.,taperanalperts=${taperanalperts},
```

Reply:
Done